docs/usage.ipynb

```diff
@@ -228,6 +229,232 @@
     }
    }
   },
+  {
```

Comment:
notebooks are hard to review

Reply:
true, probably better to review the compiled section in the docs. /-: